src/content/docs/snowflake/features/resource-monitors.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Resource Monitors"
+description: Get started with Resource Monitors in LocalStack for Snowflake
+tags: ["Base"]
+---
+
+## Introduction
+
+Resource monitors in Snowflake allow administrators to track and control credit usage for warehouses and accounts. They help manage costs by defining limits and triggering actions (such as suspending warehouses) when thresholds are reached.
+
+The Snowflake emulator offers CRUD support for resource monitors, not the actual implementation or usage of them. This is useful for testing workload governance and cost control strategies in your local development environment.
```

Comment:
> The Snowflake emulator offers CRUD support for resource monitors, not the actual implementation or usage of them.

This sentence is confusing, particularly the latter half. And the word usage here adds confusing because we say above that they track usage, so it's unclear the context. I think you mean that the created monitors will not track usage or trigger actions?